deployment/clouddeploy/gke-workers/base/backup.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: batch/v1
```

Comment:
Will need to add this file to the `resources` section of the base `kustomization.yaml` file in order for the patch in the staging instance to work.

However, that will also deploy it to production. You could copy this file into the `resources` section of `oss-vdb-test/kustomization.yaml`, and not add it to the base file.

Another option (that I haven't really tried/not sure if it works) is to deploy it to production but keep it [suspended](https://kubernetes.io/docs/concepts/workloads/controllers/cron-jobs/#schedule-suspension), with the production yaml file looking like:
```yaml
apiVersion: batch/v1
kind: CronJob
metadata:
  name: backup
spec:
  suspend: true
  jobTemplate:
    spec:
      template:
        spec:
          containers:
          - name: backup
```